database/seeds/DefaultActivityLayoutSeeder.php

```diff
@@ -20,32 +20,31 @@ public function run()
         $columnLayoutImg = 'EdE8yAybW0I4IlU8qpEZqrkIdlaou3CDcBAj1M4D.png';
         $interactiveBookImg = 'CeOAsd4QYEvpgoQpfjrVicxthAP6lM2G7LaTRFyd.png';
         $coursePresentationImg = 'rF2Vdw0bT3T7Fx85FZ7pvvZgzr0ka6DLKFLkVnVT.png';
-        $questionnaireImg = 'mtBeC6w0OY8JM4cgPetzBafE6iUPry3omWrsPS6k.png';
+        $quizImg = 'Id6BxSkmuVvZBOHNl9Wd5WUfITh4qFq2DEMO6bOJ.png';
 
         $organizations = DB::table('organizations')->pluck('id');
 
+        if (!File::exists($localURL . $InteractiveVideoImg)) {
```

Comment:
The "I" should be in lowercase i.e "interactiveVideoImg" instead of "InteractiveVideoImg".